[2021-03-17 19:29:07,214] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: taxi.SchemaGen 2021-03-17T23:28:18.372751+00:00 [queued]>
[2021-03-17 19:29:07,231] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: taxi.SchemaGen 2021-03-17T23:28:18.372751+00:00 [queued]>
[2021-03-17 19:29:07,231] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 19:29:07,231] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-17 19:29:07,231] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 19:29:07,239] {taskinstance.py:1063} INFO - Executing <Task(AirflowComponent): SchemaGen> on 2021-03-17T23:28:18.372751+00:00
[2021-03-17 19:29:07,243] {standard_task_runner.py:52} INFO - Started process 29163 to run task
[2021-03-17 19:29:07,253] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'taxi', 'SchemaGen', '2021-03-17T23:28:18.372751+00:00', '--job-id', '19', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/taxi_pipeline.py', '--cfg-path', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmp0fxdjvn1', '--error-file', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpg1goag4i']
[2021-03-17 19:29:07,256] {standard_task_runner.py:77} INFO - Job 19: Subtask SchemaGen
[2021-03-17 19:29:07,319] {logging_mixin.py:104} INFO - Running <TaskInstance: taxi.SchemaGen 2021-03-17T23:28:18.372751+00:00 [running]> on host 152.1.168.192.in-addr.arpa
[2021-03-17 19:29:07,366] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=taxi
AIRFLOW_CTX_TASK_ID=SchemaGen
AIRFLOW_CTX_EXECUTION_DATE=2021-03-17T23:28:18.372751+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-17T23:28:18.372751+00:00
[2021-03-17 19:29:07,373] {base_component_launcher.py:196} INFO - Running driver for SchemaGen
[2021-03-17 19:29:07,378] {metadata_store.py:93} INFO - MetadataStore with DB connection initialized
[2021-03-17 19:29:07,404] {base_component_launcher.py:212} INFO - Running publisher for SchemaGen
[2021-03-17 19:29:07,407] {metadata_store.py:93} INFO - MetadataStore with DB connection initialized
[2021-03-17 19:29:07,410] {python.py:118} INFO - Done. Returned value was: None
[2021-03-17 19:29:07,416] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=taxi, task_id=SchemaGen, execution_date=20210317T232818, start_date=20210317T232907, end_date=20210317T232907
[2021-03-17 19:29:07,436] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-17 19:29:07,477] {local_task_job.py:146} INFO - Task exited with return code 0
